tests/fast-integration/test/1-commerce-mock.js

```diff
@@ -39,24 +33,6 @@ function commerceMockTests(testNamespace) {
       await checkFunctionResponse(testNamespace);
     });
 
-    it('order.created.v1 legacy event should trigger the lastorder function', async function() {
-      await sendLegacyEventAndCheckResponse();
-    });
-
-    it('order.created.v1 cloud event in structured mode should trigger the lastorder function', async function() {
-      await sendCloudEventStructuredModeAndCheckResponse();
-    });
-
-    it('order.created.v1 cloud event in binary mode should trigger the lastorder function', async function() {
-      await sendCloudEventBinaryModeAndCheckResponse();
-    });
```

Comment:
Hey, are we sure we want to remove these two? I'm just checking, I remember that there was a discussion about it

Reply:
Yes, cloud eventing doesn't work in open source scenario in current Kyma